content/chat/rooms/index.textile

```diff
@@ -0,0 +1,115 @@
+---
+title: Rooms
+meta_description: "Use rooms to organize your chat traffic."
+product: chat
+languages:
+  - javascript
+---
+
+Rooms are used to organize your message traffic into logically separate 'rooms'. A room can represent a 1:1 chat between an agent and a customer, a direct message between two users in a chat room, a large group conversation, or the chat section of a livestream with thousands of users.
```

Comment:
```suggestion
Rooms are the entry object to interact with chat which provide access to most of the chat features like messages, reactions, presence, and typing indicators. A chat room can represent a 1:1 chat between an agent and a customer, a private discussion between two users in a chat application, a large group conversation, or the chat section of a livestream with thousands of users.
```
- a _direct message_ might imply a single message
- _room_ -> _application_ to avoid spreading the idea of "sub-rooms"
- attempted to rephrase first sentence